marklogic-client-api/src/test/java/com/marklogic/client/test/DatabaseClientTest.java

```diff
@@ -116,4 +120,30 @@ public void testGetClientImplementationObject() {
     assertNotNull("Client could not get client implementation", impl);
     assertTrue("", impl instanceof okhttp3.OkHttpClient);
   }
+  
+  @Test
+  public void testCheckConnectionWithValidUser() {
+		
+	DatabaseClient marklogic = DatabaseClientFactory.newClient("localhost",
+		        8000, "unittest-nodeapi", new DatabaseClientFactory.DigestAuthContext(
+				        "admin", "admin"));
+  
+    ConnectionResult connResult = marklogic.checkConnection();
+    assertTrue(connResult.isConnected());
+    assertTrue(connResult.getStatusCode() < 300);
```

Comment:
Can we remove the assertion on line 133 if the status code can be null in the successful case?